packages/date-picker/src/vaadin-date-picker-overlay-content.js

```diff
@@ -526,6 +528,11 @@ class DatePickerOverlayContent extends ControllerMixin(ThemableMixin(DirMixin(Po
 
     this._targetPosition = targetPosition;
 
+    let revealResolve;
+    this._revealPromise = new Promise((resolve) => {
```

Comment:
We could add a listener for `scroll-animation-finished` event, but using `Promise` looks like a better option. In particular, it can be also used by the `waitForScrollToFinish` test helper.